types/preview/@ember/utils/index.d.ts

```diff
@@ -12,8 +12,8 @@ declare module '@ember/utils' {
   export function isBlank(obj?: unknown): boolean;
 
   /**
-   * Verifies that a value is `null` or an empty string, empty array,
-   * or empty function.
+   * Verifies that a value is null or undefined, an empty string,
```

Comment:
We should go ahead and keep the backticks here—they affect how this gets rendered! It's the difference between these two:

> Verifies that a value is null or undefined, an empty string, or an empty array.

<!-- -->

> Verifies that a value is `null` or `undefined`, an empty string, or an empty array.

Reply:
```suggestion
   * Verifies that a value is `null` or `undefined`, an empty string,
```